Preparation/MyStack/MyStackInter.cs

```diff
@@ -1,62 +1,63 @@
 using System;
 using System.Collections;
 using System.Data;
+using System.Drawing;
 
 namespace MyStack
 {
-    public class MyStack : IEnumerable
+    public class MyStack<T>
     {
-        public static int count = 1;
-        public int topstack;
-        public int[] arrayint = new int[count];
+        private int _count = -1;
+        private T[] _underliyngArray = new T[5];
```

Comment:
why 5?

Reply:
I wrote random number. There could be any number